src/f5_tts/infer/examples/basic/basic.toml

```diff
@@ -7,4 +7,5 @@ gen_text = "I don't really care what you call me. I've been a silent spectator,
 # File with text to generate. Ignores the text above.
 gen_file = ""
 remove_silence = false
-output_dir = "tests"
\ No newline at end of file
+output_dir = "tests"
+predict_duration = false
```

Comment:
`predict_duration` -> duration_model` for clearer definition of var
same for following codes